locales/en-US/strings.properties

```diff
@@ -36,6 +36,8 @@ menu_action_open_new_window=Open in a New Window
 menu_action_open_private_window=Open in a New Private Window
 menu_action_dismiss=Dismiss
 menu_action_delete=Delete from History
+confirm_history_delete=Are you sure you want to delete every instance of this page from your history?
```

Comment:
Adding this comment again because the other got collapsed by the new pull req.
@k88hudson The design has the two texts separated. As two different paragraphs. Any way to achieve this in an a11y way?
[link to design](https://mozilla.invisionapp.com/share/49C81C6Z7#/screens/238928525_Photon_-_Confirm_Dialog)